.zenodo.json

```
@@ -1,13 +1,75 @@
 {
-    "description": "Cirq is a Python library for writing, manipulating, and optimizing quantum circuits and running them against quantum computers and simulators.",
-    "license": "Apache-2.0",
-    "title": "Cirq",
-    "upload_type": "software",
-    "creators": [
-        {
-            "name": "Cirq Developers"
-        }
-    ],
-    "access_right": "open",
-    "notes": "See full list of authors on GitHub: https://github.com/quantumlib/Cirq/graphs/contributors"
+  "title": "Cirq",
+  "description": "Cirq is a Python library for writing, manipulating, and optimizing quantum circuits and running them against quantum computers and simulators.",
+  "license": "Apache-2.0",
+  "access_right": "open",
+  "upload_type": "software",
+  "notes": "The full list of authors can be found on GitHub at the URL https://github.com/quantumlib/Cirq/graphs/contributors",
+  "creators": [
+    {
+      "name": "Cirq Developers"
+    }
+  ],
+  "related_identifiers": [
+    {
+      "identifier": "https://github.com/quantumlib/Cirq",
+      "relation": "isDerivedFrom",
+      "scheme": "url"
+    },
+    {
+      "identifier": "https://quantumai.google/cirq",
+      "relation": "isDocumentedBy",
+      "scheme": "url"
+    },
+    {
+      "identifier": "https://github.com/quantumlib/Cirq/blob/main/README.md",
+      "relation": "isDocumentedBy",
+      "scheme": "url"
+    },
+    {
+      "identifier": "https://pypi.org/project/cirq",
+      "relation": "isSupplementedBy",
+      "scheme": "url"
+    },
+    {
+      "identifier": "10.5281/zenodo.4062499",
+      "relation": "isPartOf",
+      "scheme": "doi"
+    }
+  ],
+  "keywords": [
+    "algorithms",
+    "API",
+    "application programming interface",
+    "Cirq",
+    "NISQ",
+    "noisy intermediate-scale quantum",
+    "open-source software",
+    "physics",
+    "Python",
+    "quantum algorithms",
+    "quantum circuit simulator",
+    "quantum circuit",
+    "quantum computer simulator",
+    "quantum computing",
+    "quantum development kit",
+    "quantum gate",
+    "quantum hardware modeling",
+    "quantum information science",
+    "quantum information",
+    "quantum programming language",
+    "quantum programming",
+    "quantum simulation",
+    "quantum state",
+    "quantum system",
+    "quantum theory",
+    "quantum",
+    "qubit",
+    "science",
+    "SDK",
+    "simulation",
+    "software development toolkit",
+    "software"
+  ],
+  "language": "eng",
```

The extra comma is frowned upon by `jq . .zenodo.json` and `python -m json.tool .zenodo.json`.

```suggestion
  "language": "eng"
```

---

I added that at the last minute, and apparently didn't run jsonlint afterwards, which is embarrassing.